### ISSUE TITLE
Cannot import .utm file

### ISSUE THREAD

**Issue by warvariuc** [utmapp/UTM#6699]
I cannot open a .`utm` file. "Failed to retrieve app-scope key":
```
$ /Applications/UTM.app/Contents/MacOS/UTM
2024-09-25 11:25:07.860 UTM[38856:2630526] +[IMKClient subclass]: chose IMKClient_Legacy
2024-09-25 11:25:07.860 UTM[38856:2630526] +[IMKInputSession subclass]: chose IMKInputSession_Legacy
2024-09-25 11:25:18.810 UTM[38856:2630526] [QEMULogging(0x600002d664c0)] 2024-09-25T11:25:18+0300 info com.utmapp.UTM : importing: file:///Users/vic/Documents/Linux.utm/
2024-09-25T11:25:18+0300 info com.utmapp.UTM : [UTM] importing: file:///Users/vic/Documents/Linux.utm/
2024-09-25 11:25:18.824 UTM[38856:2651192] [QEMULogging(0x600002d664c0)] 2024-09-25T11:25:18+0300 error com.utmapp.UTM : Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}
2024-09-25T11:25:18+0300 error com.utmapp.UTM : [UTM] Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}
```

**Configuration**  
* UTM Version:  Version 4.5.4 (100)
* macOS Version: 15.0 (24A335)
* Mac Chip: Apple M1 Pro

**Comment by warvariuc** [comment]
I opened the file on a laptop which wasn't upgrade to macOS Sequoia, and it worked fine. So this seems to be related to the latest macOS release.

**Comment by fiveredcars** [comment]
Same issue

**Comment by rmatinata** [comment]
Same issue.. Installed Sequoia about a week ago. Was working fine but all my VMs are suddenly grayed out. Get a bunch of these during UTM startup:

```
2024-09-28T10:44:31-0500 error com.utmapp.UTM : [UTM] Error loading 'D9D5834E-9144-49AB-9028-BA775CC01380': Error Domain=NSCocoaErrorDomain Code=257 "The file “config.plist” couldn’t be opened because you don’t have permission to view it." UserInfo={NSFilePath=/Users/***/VMs/Ubuntu Server 20.04.3.utm/config.plist, NSURL=file:///Users/***/VMs/Ubuntu%20Server%2020.04.3.utm/config.plist, NSUnderlyingError=0x600002c5a1f0 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}
```

Additionally, having trouble when creating new VMs:
```
2024-09-28 10:45:30.187 UTM[61660:10528050] [QEMULogging(0x6000037598c0)] 2024-09-28T10:45:30-0500 error com.utmapp.UTM : Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}
2024-09-28T10:45:30-0500 error com.utmapp.UTM : [UTM] Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}
```

If I re-start UTM, the new VM configuration ends up showing, but can't really import any .qcow2 or .iso into the configuration.

UTM Version: Version 4.5.4 (100)
macOS Version: 15.0 (24A335)

**Comment by warvariuc** [comment]
Works for me with the latest release [v4.6.0 (Beta)](https://github.com/utmapp/UTM/releases/tag/v4.6.0)

**Comment by warvariuc** [comment]
It worked on the first run, but fails on subsequent runs.

**Comment by mathieulassalle** [comment]
Same issue with
UTM Version: Version 4.6.0 (102)
macOS Version:  15.0.1 (24A348)
Mac Chip: Apple M1 Pro

[QEMULogging(0x600002308fc0)] 2024-10-13T11:27:41+0200 error com.utmapp.UTM : Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}

**Comment by DrBrynzo** [comment]
I haven't been able to open any VMs since upgrading to Sequoia or 15.0.1. The UTM window flashes/refreshes briefly when I drag a .utm VM or try to open one with the menu.

Console errors (filtered from lots of UTM non-error entries):

```
debug	19:28:44.902051-0700	UTM	failed to fetch /Applications/UTM.app/Contents/_CodeSignature/CodeRequirements-2 error=-10
default	19:28:44.907681-0700	UTM	[QEMULogging(0x60000363c300)] 2024-10-24T19:28:44-0700 error com.utmapp.UTM : Error Domain=NSCocoaErrorDomain Code=256 "Failed to retrieve app-scope key" UserInfo={NSDebugDescription=Failed to retrieve app-scope key}
debug	19:28:44.907752-0700	UTM	Bundle: <private>, key: Err256, value: (null), table: FoundationErrors, localizationNames: (null), result: The file couldn’t be opened.
error	19:28:45.222548-0700	UTM	6 Reentrant message: kDragIPCCompleted, current message: kDragIPCLeaveApplication


```

**Comment by devon-berta-aquia** [comment]
Maybe fixed in sequoia 15.1 per this bug report:
https://forums.developer.apple.com/forums/thread/764435?answerId=806097022#806097022

Upgrading now and testing. Will share results.

**Comment by devon-berta-aquia** [comment]
I can confirm sequoia 15.1 with UTM 4.5.4 version on M3 Mac is working again.
